content/docs/user-guide/components/reference/vegetation/vegetation-area-system.md

```diff
@@ -0,0 +1,73 @@
+---
+linkTitle: Vegetation Area System
+title: Vegetation Area System Component
+description: The Vegetation Area System component handles the placement and removal of vegetation instances based on its configured rules in Open 3D Engine (O3DE).
+---
+
+The **Vegetation Area System** component handles the placement and removal of vegetation instances throughout your world, based on its configured rules.
+
+Learn about how the Vegetation Area System component works within the vegetation system in [Technical details](#technical-details) below.
+
+{{< note >}}
+This is a system component that exists when you interact with the vegetation system via the Vegetation Gem. Generally, this system is already set up and you don't need to do anything further to manage vegetation in your world.
+{{< /note >}}
+
+## Provider
+
+[Vegetation Gem](/docs/user-guide/gems/reference/environment/vegetation/)
+
+## Source code
+
+[`/Gems/Vegetation/Code/Source/AreaSystemComponent.h`](https://github.com/o3de/o3de/blob/development/Gems/Vegetation/Code/Source/AreaSystemComponent.h)
+
+## Parameters
+
+| Property | Description | Values | Default |
+| --- | --- | --- | --- |
+| **View Area Grid Size** | The number of sectors (per-side) of a managed grid in a scrolling view centered around the camera | 1 to 128 | 13  |
+| **Sector Point Density** | The number of equally spaced vegetation instance grid placement points (per-side) within a sector | 1 to 64 | 20 |
+| **Sector Size In Meters** | The size in meters per-side of each sector. | 1 to 1024 | 16 |
+| **Thread Processing Interval** | The delay in milliseconds between processing queued thread tasks. | 0 to 5000 | 500 |
+| **Sector Search Padding** | Increases the search radius for surrounding sectors when enumerating through vegetation instances. | 0 to 2 | 0 |
+| **Sector Point Snap Mode** | Controls whether vegetation placement points are located at the corner or the center of the cell. | `Corner`, `Center` | `Center` |
```

Comment:
```suggestion
| **Sector Point Snap Mode** | Controls whether vegetation placement points are located at the corner or the center of the cell. | `Corner`, `Center` | `Corner` |
```
You should double-check this, in the _level component_, Vegetation System Settings, this defaults to `Corner`.

Reply:
Checks out in code too, thanks!